HelpSource/Classes/Dictionary.schelp

```diff
@@ -193,10 +193,22 @@ d.matchAt(\abc)
 ::
 
 method::trueAt
-Returns link::Classes/True:: if the item at the key is true, otherwise false. This method is also valid in link::Classes/Object::.
+Returns link::Classes/True:: if the item's booleanValue at the key is true, otherwise false. This method is also valid in link::Classes/Object::.
```

Comment:
See my comment in the Conversation thread regarding the inconsistent behavior with `Object:-trueAt`

Reply:
Given the divergence in behavior with `Object:-falseAt`, I think we should just remove this reference to implied parallel behavior with `Object`:
`This method is also valid in link::Classes/Object::.`